ethereum/contracts/Starport.sol

```diff
@@ -467,4 +478,8 @@ contract Starport {
         }
         return abi.decode(_returnData, (string)); // All that remains is the revert string
     }
+
+    function toBytes32(address addr) public pure returns (bytes32) {
```

Comment:
Find the whole list of interesting methods for conversion https://ethereum.stackexchange.com/questions/884/how-to-convert-an-address-to-bytes-in-solidity

Reply:
Yeah, none of them really worked well for me, but this one did and it's the most obvious.